application/src/main/java/org/togetherjava/tjbot/commands/help/HelpSystemHelper.java

```diff
@@ -274,6 +274,10 @@ private void executeUncategorizedAdviceCheck(long threadChannelId, long authorId
                 return new CompletedRestAction<>(jda, null);
             }
 
+            if (threadChannel.isArchived()) {
+                return new CompletedRestAction<>(jda, null);
```

Comment:
please add a debug-level log, similar to the one above. otherwise its hard for us to debug.